docs/contributing/doc_guide.rst

```diff
@@ -497,6 +497,42 @@ example, ``:cite:p:`wilson:2014``` will show up as :cite:p:`wilson:2014`,
 ``:cite:p:`wilson:2014, wilson:2017``` will show up as
 :cite:p:`wilson:2014, wilson:2017`.
 
+.. _api-static:
+
+Creating a documentation stub file for a new module
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Would be awesome to add an example error that pops up when this is broken, so it's searchable.